alacritty/src/event.rs

```diff
@@ -430,7 +433,10 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
         }
 
         // Move vi cursor down if resize will pull from history.
-        if self.terminal.history_size() != 0 && self.terminal.grid().display_offset() == 0 {
+        if self.terminal.history_size() != 0
```

Comment:
It seems like tails of `confirm_search` and `cancel_search` are exactly the same, I'd suggest merging them, since the tail is basically hiding the search.